swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -39,20 +39,9 @@ Follow the instructions given in this section to develop the service.
 
     ![Open diagram view](/learn/images/integration-tutorials/sending-a-message-to-a-service/open_diagram_view.gif)
 
-3. Define two [configurable variables](https://ballerina.io/learn/by-example/#configurability) for the port on which the listener should listen and the URL of the backend service.
+3. Define a record type named `Doctor` corresponding to the payload from the backend service.
```

Comment:
I don't think this is what @sameerajayasoma meant in Suggestion 2. We can describe how using records help with data validation, access, etc.